src/SchedulerSubsystem/Scheduler.java

```diff
@@ -8,30 +8,46 @@
 import Networking.Transmitters.DMA_Transmitter;
 import com.sun.jdi.InvalidTypeException;
 
+import java.util.*;
+
 public class Scheduler implements Runnable {
     private final DMA_Transmitter transmitterToFloor;
     private final DMA_Transmitter transmitterToElevator;
     private final DMA_Receiver receiver;
+    private Set<DestinationEvent> floorRequests;
 
     public Scheduler(DMA_Receiver receiver, DMA_Transmitter transmitterToFloor, DMA_Transmitter transmitterToElevator) {
         this.receiver = receiver;
         this.transmitterToElevator = transmitterToElevator;
         this.transmitterToFloor = transmitterToFloor;
+        floorRequests = new HashSet<DestinationEvent>();
     }
 
     /**
-     * process event received from the floor (up or down request)
-     * and sends it to the elevator.
-     * @param destinationEvent a destination event
+     * Process event received from the floor.
+     *
+     * @param destinationEvent Destination event received from floor.
      */
     private void processDestinationEvent(DestinationEvent destinationEvent) {
-        transmitterToElevator.send(destinationEvent);
-
+        // Store event locally to use in scheduling
+        floorRequests.add(destinationEvent);
     }
 
-    private boolean isElevatorStopping(ElevatorStateEvent event) {
-        // TODO: Implement check
-        return true;
+    /**
+     * Returns true if elevator should stop, false otherwise.
+     *
+     * @param e Elevator state to check.
+     * @return True if elevator should stop, false otherwise.
+     */
+    public boolean isElevatorStopping(ElevatorStateEvent e) {
```

Comment:
I'm still not a fan of keeping this public but we can talk more on Tuesday when we discuss testing.